src/Conda.jl

```diff
@@ -137,34 +191,40 @@ function _install_conda(force=false)
             end
         end
         Conda.add_channel("defaults")
+        # Update conda because conda 4.0 is needed and miniconda download installs only 3.9
+        run(_set_conda_env(`$(Conda.conda) update -y conda`))
+    end
+    if !isdir(prefix(env))
```

Comment:
`prefix` already does if it is a full path. In case of a environment name, the environment has to be created here.